src/Elmish.WPF/WpfProgram.fs

```diff
@@ -21,21 +21,32 @@ type WpfProgram<'model, 'msg> =
 module WpfProgram =
 
 
-  let private create getBindings program =
+  let private create createViewModel program =
     { ElmishProgram = program
-      Bindings = getBindings ()
+      CreateViewModel = createViewModel
       LoggerFactory = NullLoggerFactory.Instance
       ErrorHandler = fun _ _ -> ()
       PerformanceLogThreshold = 1 }
 
+  let createDynamic (getBindings: unit -> Binding<'model, 'msg> list) performanceLogThreshold bindingsLogger performanceLogger : ('model * ('msg -> unit) -> DynamicViewModel<'model, 'msg>) =
+    let bindings = getBindings ()
+    fun (initialModel, dispatch) -> DynamicViewModel<'model, 'msg>(initialModel, dispatch, bindings, performanceLogThreshold, "main", bindingsLogger, performanceLogger)
```

Comment:
This function illustrates (in a somewhat hacky way) the way we can fairly easily transpose form `getBindings: unit -> Binding<'model, 'msg>` into a `CreateViewModel: 'model * ('msg -> unit) -> 'viewModel` paradigm. This isn't easily reversible though without reflection.